components/inputs/input-text.js

```diff
@@ -461,8 +466,13 @@ class InputText extends FocusMixin(LabelledMixin(FormElementMixin(SkeletonMixin(
 		}
 
 		let tooltip = nothing;
-		if (this.validationError && !this.skeleton && !this.noValidate) {
-			tooltip = html`<d2l-tooltip state="error" announced align="start">${this.validationError} <span class="d2l-offscreen">${this.description}</span></d2l-tooltip>`;
+		if (!this.skeleton) {
+			if (this.validationError && !this.noValidate) {
+				// this tooltip is using "announced" since we don't want aria-describedby wire-up which would bury the message in VoiceOver's More Content Available menu
+				tooltip = html`<d2l-tooltip state="error" announced align="start">${this.validationError} <span class="d2l-offscreen">${this.description}</span></d2l-tooltip>`;
+			} else if (this.controlInstructions) {
+				tooltip = html`<d2l-tooltip align="start" for="${this._inputId}" delay="1000">${this.controlInstructions}</d2l-tooltip>`;
```

Comment:
Unlike the error `d2l-tooltip` above, the offscreen `description` is not included in this tooltip since our intention is that it `description` will be removed from `d2l-input-text`. I've left that `description` in place because removing `description` will be a separate set of work. 

Also, we do _not_ use `announced` here because we want direct tooltip wire-up using `aria-describedby` (we're "ok" with these control instructions being buried in VoiceOver's "More Content Available" menu.